src/SFA.DAS.ProviderApprenticeshipsService.Web/Extensions/HtmlHelperExtensions.cs

```diff
@@ -83,5 +84,48 @@ public static MvcHtmlString GetClientContentByType(this HtmlHelper html, string
             var content = userResponse;
             return MvcHtmlString.Create(content.Content);
         }
+
+        public static MvcHtmlString SetZendeskLabels(this HtmlHelper html, params string[] labels)
+        {
+            var apiCallString = "<script type=\"text/javascript\">zE('webWidget', 'helpCenter:setSuggestions', { labels: [";
+
+            var first = true;
+            foreach (var label in labels)
```

Comment:
I think the code implemented in EAS is a bit cleaner, and might be good to keep it consistent:
https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/blob/a59dd8ffbe9f630bd26aff947fdad96a9f5fd23a/src/SFA.DAS.EmployerAccounts.Web/Extensions/HtmlHelperExtensions.cs#L51-L60

Might be worth updating the confluence page with the alternate approach

Reply:
Changed to EAS version